opm/output/data/Wells.hpp

```diff
@@ -452,11 +480,44 @@ namespace Opm {
             case opt::well_potential_gas: return this->well_potential_gas;
             case opt::brine: return this->brine;
             case opt::alq: return this->alq;
+            case opt::tracer: return this->tracer.at("");
+        }
+
+        throw std::invalid_argument(
+                "Unknown value type '"
+                + std::to_string( static_cast< enum_size >( m ) )
+                + "'" );
+
+    }
+
+    inline const double& Rates::get_ref( opt m, const std::string& tracer_name ) const {
+        switch( m ) {
+            case opt::tracer: return this->tracer.at(tracer_name);
+            case opt::wat: return this->wat;
```

Comment:
I would assume it was a programmer bug to call this function with `m != opt::tracer`- i.e. the implementation could be something like:
```C++
if (m != opt::tracer)
   throw std::logic_error("Logic error - should be called with tracer argument");

return this->tracer.at( tracer_name );
```

Reply:
Agree.